.mockery.yaml

```diff
@@ -9,3 +9,4 @@ packages:
   github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/ipaccesslist:
   github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/dbuser:
   github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/deployment:
+  github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/customroles:
```

Comment:
The mocks are missing, you will need to run mockery manually for now, we have no integration in the Makefile for that yet.